src/helpers/useParaSwapTransactionHandler.tsx

```diff
@@ -77,6 +83,7 @@ export const useParaSwapTransactionHandler = ({
     asset: string;
     amount: string;
   }
+  // @ts-expect-error TODO: need think about "tx" type
```

Comment:
i think some types errors were introduced with the 18 ugprade specifically on paraswap